torchvision/models/quantization/mobilenetv2.py

```diff
@@ -46,6 +46,8 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         Args:
            Inherits args from floating point MobileNetV2
         """
+        if "init_weights" not in kwargs:
+            kwargs["init_weights"] = True
```

Comment:
MobileNetV2 does not accept `init_weights` as argument, right ? Let's revert this change.